samples/nrf5340/smp_svr/sample.yaml

```diff
@@ -0,0 +1,14 @@
+sample:
+  description: Simple Management Protocol sample
+  name: smp svr
+common:
+  harness: bluetooth
```

Comment:
consider changing it to at least harness: console, to just verify if sample is booted,
you can use something like that:
```
  harness: console
  harness_config:
    type: one_line
    regex:
      - "Hello World! (.*)"
  ```

Reply:
or I can later (in separate PR) extend it with a pytest scenario to test full upgrade with `mcumgr` (like in https://github.com/zephyrproject-rtos/zephyr/tree/main/tests/boot/with_mcumgr)